docs/csharp/programming-guide/file-system/how-to-create-a-key-in-the-registry.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: "How to: Create a Key In the Registry (Visual C#)"
+title: "How to create a key in the registry - C# Programming Guide"
```

Comment:
Here, I changed `(Visual C#)` to ` - C# Programming Guide` to be consistent with the other articles.